### ISSUE TITLE
Crash on first click

### ISSUE THREAD

**Issue by slaanx** [PippleCultist/HoloCureMultiplayerMod#16]
I get this error when I click on the first splash screen. I have it installed on my second drive. I'm playing the steam version.

```
___________________________________________
############################################################################################
ERROR in
action number 1
of Create Event
for object obj_TitleScreen:

Creating instance for non-existing object: -1
0> CallbackManagerMod!_AurieFrameworkInit+0x2D056
1> HoloCure+0x22308B9
2> HoloCure+0x21A73A1
3> YYToolkit!ModuleUnload+0x61AE
4> YYToolkit!ModuleUnload+0x5EDD
5> HolocureMultiplayerMod!_AurieFrameworkInit+0x26D72
6> CallbackManagerMod!_AurieFrameworkInit+0x2721
7> YYToolkit!ModuleUnload+0x1D4BB
8> YYToolkit!ModuleUnload+0x1C685
9> HoloCure+0x2158963
10> HoloCure+0x22CA8F5
11> HoloCure+0x22B1BFB
12> HoloCure+0x220ACFE
13> HoloCure+0x2208371
14> HoloCure+0x2209BE2
15> HoloCure+0x22363EB
16> HoloCure+0x242542E
17> KERNEL32!BaseThreadInitThunk+0x1D
18> ntdll!RtlUserThreadStart+0x28
############################################################################################
```

**Comment by PippleCultist** [comment]
Can you check the Logs folder in your HoloCure game directory and post the logs there as well? It's possible that you might have conflicting versions of some mods if you've used some of my other mods, so check to make sure that you're using all the mods in the current release.

**Comment by PippleCultist** [comment]
Also, I just remembered that another person had the same issue, and it was due to windows defender blocking AurieLoader and not letting it install properly. Try reinstalling Aurie/making a whitelist in windows defender for AurieLoader

**Comment by slaanx** [comment]
Here is the log:
2024_10_28_00_52_59 Holocure Menu Mod v1.0.2 - Finished initializing
2024_10_28_00_53_02  - Creating instance for non-existing object: -1
0> CallbackManagerMod!_AurieFrameworkInit+0x2D056
1> HoloCure+0x22308B9
2> HoloCure+0x21A73A1
3> YYToolkit!ModuleUnload+0x61AE
4> YYToolkit!ModuleUnload+0x5EDD
5> HolocureMultiplayerMod!_AurieFrameworkInit+0x26D72
6> CallbackManagerMod!_AurieFrameworkInit+0x2721
7> YYToolkit!ModuleUnload+0x1D4BB
8> YYToolkit!ModuleUnload+0x1C685
9> HoloCure+0x2158963
10> HoloCure+0x22CA8F5
11> HoloCure+0x22B1BFB
12> HoloCure+0x220ACFE
13> HoloCure+0x2208371
14> HoloCure+0x2209BE2
15> HoloCure+0x22363EB
16> HoloCure+0x242542E
17> KERNEL32!BaseThreadInitThunk+0x1D
18> ntdll!RtlUserThreadStart+0x28

2024_10_28_00_53_05 Holocure Multiplayer Mod v1.1.7 - Finished ModuleInitialize

**Comment by slaanx** [comment]
Not sure if it matters, but AurieManager is in the same folder as Holocure.

**Comment by slaanx** [comment]
All the mods I'm using are from the latest release 1.1.7

**Comment by slaanx** [comment]
I was able to get it to run, but it was purely brute force. I learned that if I click as soon as allowed to skip the opening cutscene it will immediately crash, but if I wait a moment it will work.

I noticed that if I click before the YYToolkit Log says "Start initializing" it will crash. But if I wait till after, it won't.

**Comment by slaanx** [comment]
The YYToolkit Log window reports a ton of errors, but it's not producing a log file.

**Comment by PippleCultist** [comment]
Those aren't important issues. Is the mod working now, or is it still crashing? Try launching the game through steam instead of AurieManager since there's some issues with that currently. If you're still getting that issue, you might just have to try reinstalling multiple times/turning off Windows Defender since it might be messing up the Aurie initialization.

**Comment by slaanx** [comment]
1. You are correct, with or without your mod installed, running the game through AurieManager is very unreliable and is rarely able to actually run the game. Running it via steam is 100% reliable.
2. Running the game through steam with Aurie installed (without your mod ) apparently works. I ran through a whole level without any issues.
3. If I run through steam with your mod installed and click before the YYToolkit Log says "Start Initializing" It crashes, everytime. with the error I reported above.
4. Alternatively, if I wait to click it loads into the menu. I played a full level with your mod active and it seems to work fine.
5. I have yet to test the mod in multiplayer, however. For now, the game works in single-player while your mod is installed.
